.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: Bug Report
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
```

Comment:
Could add a hint to use a pastbin for long log:
```suggestion
<!--- HINT: You can paste gist.github.com links for long logs or larger files -->
```

Reply:
Done as desired